### ISSUE TITLE
Undocumented times argument is required for live transcription

### ISSUE THREAD

**Issue by MasterOdin** [deepgram/deepgram-node-sdk#109]
## What is the current behavior?

I'm getting the following type error when I try to use `deepgram.transcription.live` following the [transcribe sdk docs](https://developers.deepgram.com/sdks-tools/sdks/node-sdk/streaming-transcription/#initiating-a-connection):

```
Property 'times' is missing in type '{ interim_results: true; punctuate: true; }' but required in type 'LiveTranscriptionOptions'.
```

Looking at the [API docs](https://developers.deepgram.com/api-reference/transcription/#transcribe-live-streaming-audio), any `times` argument is not mentioned at all.

## Steps to reproduce

Have the following code with TS:

```ts
import { Deepgram } from '@deepgram/sdk';

const deepgram = new Deepgram("API_KEY");
const socket = deepgram.transcription.live({
  punctuate: true,
});
```

## Expected behavior

That I wouldn't get a TS error.

## Please tell us about your environment

> We want to make sure the problem isn't specific to your operating system or programming language.
  
- **Operating System/Version:** macOS
- **Language:** TypeScript
- **Browser:** Chrome

## Other information

I'm using `@deepgram/sdk@1.17.0`. This is the TS type that's throwing an error:

https://github.com/deepgram/deepgram-node-sdk/blob/7821a6d573b3e1a068607273973b7d9e38269962/src/types/liveTranscriptionOptions.ts#L210-L214

Given how everything else is optional in this file, I feel like this should be to?

**Comment by briancbarrow** [comment]
@MasterOdin Thanks for point this out. We'll get a fix out today.

**Comment by briancbarrow** [comment]
Fixed in #110
